src/components/component-renderer/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Paragraph from './paragraph';
```

Comment:
For easier developer experience is it worth mimicking the elements from HTML? For example instead of a `Paragraph` component, we just call it `P` (with a capital). This way users with prior understanding of html know what they can use.